cli/analytics.go

```diff
@@ -0,0 +1,42 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"github.com/bitrise-io/bitrise/tools/filterwriter"
+	"github.com/bitrise-io/envman/env"
+	"github.com/bitrise-io/envman/models"
+)
+
+func expandStepInputsForAnalytics(environment, inputs []models.EnvironmentItemModel, secrets []string) (map[string]string, error) {
+	sideEffects, err := env.GetDeclarationsSideEffects(environment, &env.DefaultEnvironmentSource{})
+	if err != nil {
+		return map[string]string{}, fmt.Errorf("expandStepInputsForAnalytics() failed, %s", err)
+	}
+
+	// Filter inputs from enviroments
+	expandedInputs := make(map[string]string)
+	for _, input := range inputs {
+		inputKey, _, err := input.GetKeyValuePair()
+		if err != nil {
+			return map[string]string{}, fmt.Errorf("expandStepInputsForAnalytics() failed, %s", err)
```

Comment:
pls use error message which describes the issue better, and uses half-sentences `failed to get input key: %s`, input.GetKeyValuePair() returns error messages ,like: `more than 1 environment key specified: ...`

Reply:
pls check all the new error messages